src/dialog/overflow-menu/overflow-menu-option.component.ts

```diff
@@ -72,7 +93,7 @@ export class OverflowMenuOption implements AfterViewInit {
 	}
 
 	ngAfterViewInit() {
-		const button = this.elementRef.nativeElement.querySelector("button");
+		const button = this.elementRef.nativeElement.querySelector("button , a");
 		if (button.scrollWidth > button.offsetWidth) {
 			this.title = this.elementRef.nativeElement.querySelector("button").textContent;
```

Comment:
I'm not sure why we ended up with two of the same `querySelectors` ... this should just be 
```suggestion
			this.title = button.textContent;
```

Not your fault, just a weird artifact of something 🤔